src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -35,9 +35,8 @@ redirects:
   - /docs/mobile-monitoring/new-relic-mobile-android/install-configure
 ---
 
-Our [guided install](https://onenr.io/0kjnEoryzRo) is the preferred method of installation, but if you need to manually install and configure our Android agent, follow the instructions below.
 
-If you haven't already, create your free New Relic account below to start monitoring your data today.
+We recommend that you use our [guided install](https://onenr.io/0kjnEoryzRo) to set up Android monitoring. However, if you need to manually install for any reason, follow the steps below to install the New Relic Android agent with Gradle and Android Studio.
```

Comment:
I would remove the "for any reason" as it seems unneeded, but it's just a small thing to consider.